integration-tests/oauth2-reactive-spring/src/test/resources/testRunner.yml

```diff
@@ -0,0 +1,80 @@
+      - --server.servlet.session.tracking-modes=cookie
+      - --debug
+    protectedPath: /oauth2/authorization/okta
+
+  implicit-flow-local-validation:
+    command: com.okta.spring.tests.common.reactive.implicit.ReactiveImplicitFlowApplication
+    args:
+      - --server.port=${applicationPort}
+      - --spring.security.oauth2.resourceserver.jwt.issuer-uri=https://localhost:${mockHttpsPort}/oauth2/default
+      - --spring.security.oauth2.resourceserver.jwt.jwk-set-uri=https://localhost:${mockHttpsPort}/oauth2/default/v1/keys
+      # A client ID is NOT required, #218
+      - --server.session.trackingModes=cookie
+      - --debug
+
+  oidc-code-flow-local-validation:
+    disabledTests:
+      # Spring does remote validation of the token
+      - invalidIssuerIdTokenJwtTest
+      # Fix for https://github.com/spring-projects/spring-security/issues/5562 && 6484
+      - noAuthCodeTest
+      - wrongStateTest
+    command: com.okta.spring.tests.common.reactive.code.ReactiveRedirectCodeFlowApplication
+    args:
+      - --server.port=${applicationPort}
+      - --server.servlet.session.tracking-modes=cookie
+      - --spring.security.oauth2.resourceserver.jwt.issuer-uri=https://localhost:${mockHttpsPort}/oauth2/default
+      - --spring.security.oauth2.client.provider.okta.authorization-uri=https://localhost:${mockHttpsPort}/oauth2/default/v1/authorize
+      - --spring.security.oauth2.client.provider.okta.issuer-uri=https://localhost:${mockHttpsPort}/oauth2/default
+      - --spring.security.oauth2.client.registration.okta.client-id=OOICU812
+      - --spring.security.oauth2.client.registration.okta.client-secret=VERY_SECRET
+      # not implemented until Spring Sec 5.2
+      #- --spring.security.oauth2.client.registration.okta.redirect-uri={baseUrl}/authorization-code/callback
+      - --spring.security.oauth2.client.registration.okta.scope=profile,email,openid
+      - --okta.oauth2.postLogoutRedirectUri=/you-are-logged-out
+    protectedPath: /oauth2/authorization/okta
+
+  pkce-code-flow-remote-validation:
+    command: com.okta.spring.tests.common.reactive.code.ReactiveRedirectCodeFlowApplication
+    args:
+      - --server.port=${applicationPort}
+      - --server.servlet.session.tracking-modes=cookie
+      - --spring.security.oauth2.resourceserver.jwt.issuer-uri=https://localhost:${mockHttpsPort}/oauth2/default
+      - --spring.security.oauth2.client.provider.okta.authorization-uri=https://localhost:${mockHttpsPort}/oauth2/default/v1/authorize
+      - --spring.security.oauth2.client.provider.okta.issuer-uri=https://localhost:${mockHttpsPort}/oauth2/default
+      - --spring.security.oauth2.client.registration.okta.client-id=OOICU812
+      - --spring.security.oauth2.client.registration.okta.scope=offline_access
+      # not implemented until Spring Sec 5.2
+      #- --spring.security.oauth2.client.registration.okta.redirect-uri={baseUrl}/authorization-code/callback
```

Comment:
Double-check these "Spring Sec 5.2" comments, they should be resolved or removed (I'm guessing the redirect URI has been set in the related application.properties files?

Reply:
I've uncommented this.